Zend/zend_inheritance.c

```diff
@@ -619,9 +619,7 @@ static void do_inheritance_check_on_method(zend_function *child, zend_function *
 		zend_string *method_prototype = zend_get_function_declaration(parent);
 		zend_string *child_prototype = zend_get_function_declaration(child);
 
-		if (child->common.prototype && (
-			child->common.prototype->common.fn_flags & ZEND_ACC_ABSTRACT
-		)) {
+		if (parent->common.fn_flags & ZEND_ACC_ABSTRACT) {
```

Comment:
@pmmaga Correct. However I don't see a great way of implementing that, short of going two LSP checks, one against the interface and one against the class. Not sure if we want to do that...